api/types/role.go

```diff
@@ -790,6 +790,14 @@ func (r *RoleV6) CheckAndSetDefaults() error {
 	if r.Spec.Options.SSHFileCopy == nil {
 		r.Spec.Options.SSHFileCopy = NewBoolOption(true)
 	}
+	if r.Spec.Options.IDP == nil {
+		// By default, don't allow users to access the IdP.
+		r.Spec.Options.IDP = &IdPOptions{
+			SAML: &IdPSAMLOptions{
+				Enabled: NewBoolOption(false),
+			},
+		}
```

Comment:
The RFD says
```
Note: users will default to having IdP access enabled unless administrators specify otherwise.
```

So, I was expecting a `Enabled: true` here.

Reply:
Ahhh yeah, sorry. I argued myself the opposite way for some reason... will change!